data/css/preset_a.scss

```diff
@@ -22,77 +29,88 @@
 
 .set-page,
 .search-page {
-    padding: 0 100px;
-
     .BannerSet .Card__title,
     .BannerSearch__title {
-        padding-top: 1.0em;
-        padding-bottom: 1.0em;
-
-        font-size: 36px;
+        color: white;
+        font-size: 55px;
+        padding: 1.0em 1.0em 0.50em 1.0em;
+        text-shadow: 2px 8px 6px transparentize(black, 1 - 0.20);
     }
 
-    separator {
-        min-height: 1px;
-        background-color: rgba(255, 255, 255, 0.3);
+    .CardLegacyPolaroid {
+        padding: 15px;
+        box-shadow: 2px 8px 5px transparentize(black, 1 - 0.20);
+        &__title {
+            font-size: 1.0em;
+        }
+
+        &__synopsis {
+            font-size: 0.75em;
+        }
     }
 
-    .CardLegacyPolaroid__title {
-        font-size: 0.77em;
-        text-shadow: 0px 1px 0px white;
+    .ContentGroupContentGroup {
+        padding: 0px 100px 70px 100px;
     }
 }
 
-.set-page flowbox, .search-page flowbox {
-    margin-top: 20px;
+.set-page flowbox, 
```

Comment:
Trailing white space here, please remove it (and all others). Usually, you can see these white spaces with ``` git log -p ```